### PR TITLE
Described what buttons to use for a Zettelkasten

### PR DESCRIPTION
**Pull request by Vetii**
It took me a while to realize that the "identifiers" described were inside the file and not the generated file names, and that I can name my notes however I want. It also took me quite a while to find the `Ctrl + L` shortcut, since it's not described in the Zettelkasten article. Since I assumed people would want this information to be in this article, I added them.

### CODE REVIEW

File: docs/en/guides/guide-zettelkasten.md [hunk 1]
```diff
@@ -30,4 +30,9 @@ To add a new directory to Zettlr, hit `Cmd/Ctrl+O`, click the first toolbar butt
 
 ## Step 3: Write!
 
-Now you'll be set to use Zettlr as your Zettelkasten management system. If you want to dig deeper into how a Zettelkasten should work and what it should look like, consult the immense amount of resources on Zettelkästen, available on the web. A good starting point is [Zettelkasten.de](https://www.zettelkasten.de/). The authors have assembled a large list of (english) blog posts on how to supercharge your writing using a Zettelkasten.
+Now you'll be set to use Zettlr as your Zettelkasten management system. Two operations are important in a Zettelkasten:
+
+1. Creating an identifier to a note, so that you can link to it: You can do this by pressing `Ctrl + L` while editing any note, which will paste a generated ID to your note.
```
Comment (ryota-abe) [inline docs/en/guides/guide-zettelkasten.md]:
```suggestion
1. Creating an identifier to a note, so that you can link to it: You can do this by pressing `Cmd/Ctrl+L` while editing any note, which will paste a generated ID to your note.
```
For consistency with other pages.

Comment (nathanlesage) [reply docs/en/guides/guide-zettelkasten.md]: Yes, I agree!

However, as correctly mentioned in the commit, it would be good to  also add the information afterwards, that the ID in fact can also be put in the filename, which is why the default value for the filename is set to the ID!

File: docs/en/guides/guide-zettelkasten.md [hunk 2]
```diff
@@ -30,4 +30,9 @@ To add a new directory to Zettlr, hit `Cmd/Ctrl+O`, click the first toolbar butt
...
+2. Linking to other notes: Pressing `[[` while editing will display a list of notes you can link to (those who have an idenfitier in them).
```
Comment (nathanlesage) [inline docs/en/guides/guide-zettelkasten.md]: This is not entirely true, as the users have the ability to change the links (see for this the Zettelkasten preferences), which is why I would rephrase this to something like

> Linking to other notes: Insert the link opening (default: `[[`) while editing to display the auto-complete dropdown, from which you can choose the file to link to.

Furthermore the addendum "those who have an identifier in them" is wrong, because you can also link to files without any ID — only in this case via filename instead of the identifier.

Comment (nathanlesage) [inline docs/en/guides/guide-zettelkasten.md]: This is not entirely true, as the users have the ability to change the links (see for this the Zettelkasten preferences), which is why I would rephrase this to something like

> Linking to other notes: Insert the link opening (default: `[[`) while editing to display the auto-complete dropdown, from which you can choose the file to link to.

Furthermore the addendum "those who have an identifier in them" is wrong, because you can also link to files without any ID — only in this case via filename instead of the identifier.

Comment (nathanlesage) [inline docs/en/guides/guide-zettelkasten.md]: This is not entirely true, as the users have the ability to change the links (see for this the Zettelkasten preferences), which is why I would rephrase this to something like

> Linking to other notes: Insert the link opening (default: `[[`) while editing to display the auto-complete dropdown, from which you can choose the file to link to.

Furthermore the addendum "those who have an identifier in them" is wrong, because you can also link to files without any ID — only in this case via filename instead of the identifier.

**Comment by nathanlesage** [comment]
Please excuse the duplicate comments, the GitHub API seems to be degraded currently, had problems submitting the reports!

File: docs/en/guides/guide-zettelkasten.md (hunk 2 above)
Comment (Vetii) [reply docs/en/guides/guide-zettelkasten.md]: I am making the changes, one question though: Is there any difference between a link to a note with an ID and a link to a note without?

Comment (nathanlesage) [reply docs/en/guides/guide-zettelkasten.md]: Except from exactly that: no! IDs are just easier because this de-couples the content from any file name, so you can rename the file and the links will still work. If you don't plan on renaming your files, then you could live 100% without IDs!

Comment (Vetii) [reply docs/en/guides/guide-zettelkasten.md]: Ok thanks!

Pushed the changes you requested :)

**Comment by nathanlesage** [comment]
Hey, I completely forgot about this PR! Sorry about that and the delay! :see_no_evil:
